docs/pages/admin-guides/access-controls/access-requests/resource-requests.mdx

```diff
@@ -406,6 +406,72 @@ Requesting access to a Kubernetes Namespace allows you to access all resources
 in that namespace but you won't be able to access any other supported resources
 in the cluster.
 
+##### Enforce users to request only allowed Kubernetes Resources
+
+The `request.kubernetes_resources` field allows you to restrict what kinds of Kubernetes 
+resources a user can access request to.
+
+The following role assigned to a user, restricts access requesting to only Kubernetes namespaces.
+If a user attempts to make a request to Kubernetes resources other than `namespace` then the request
+will be rejected:
+
+```yaml
+kind: role
+metadata:
+  name: requester-kube-access
+version: v7
+spec:
+  allow:
+    request:
+      search_as_roles:
+        - "kube-access"
+      kubernetes_resources:
+        - kind: "namespace"
+  deny: {}
+```
+
+The following role assigned to a user, restricts access requesting to only Kubernetes namespaces and/or pods.
+
+```yaml
+kind: role
+metadata:
+  name: requester-kube-access
+version: v7
+spec:
+  allow:
+    request:
+      search_as_roles:
+        - "kube-access"
+      kubernetes_resources:
+        - kind: "namespace"
+        - kind: "pod"
+  deny: {}
+```
+
+The following role assigned to a user, restricts users from access requesting to a Kubernetes cluster but
+allows requesting to any Kubernetes resources:
```

Comment:
```suggestion
The following role restricts users from requesting access to a Kubernetes cluster but
allows requesting to any Kubernetes resources:
```

- Remove "assigned to a user." The sentence reads just as well without it.
- "Access requesting" is not a verb. Let's say "requesting access" instead.
- I don't quite understand the example - how does it restrict users from requesting access to kube clusters?

Reply:
i removed the `restricting users from requesting access to kube cluster` phrase from this example. instead i moved it to the beginning of section like this:

```
The `request.kubernetes_resources` field allows you to restrict what kinds of Kubernetes 
resources a user can request access to. Configuring this field to any value will disallow
requesting access to the entire Kubernetes cluster.

If the `request.kubernetes_resources` field is not configured, then a user can request access 
to any Kubernetes resources, including the entire Kubernetes cluster.
```

does this make more sense?

Reply:
Yep!